### ISSUE TITLE
Random view ordering when first field in order is unset

### ISSUE THREAD

**Issue by drn05r** [eprints/eprints3.4#105]
If you want to order a view by a field that is only set for some items rather than it using the second order field to order by if the first is not set the order appears to be random.

**Comment by drn05r** [comment]
There does seem to be some quirky behaviour with freetext order where things like the indefinite article gets moved to the end of the title when indexed but otherwise if you have a browse view say with ISSN followed by title as the ordering criteria,  If none of the items have an ISSN or all of the items have the same ISSN (as an example), the ordering will be the same.  However, items without an ISSN will be ordered before those with an ISSN. 

In conclusion, although there are quirks, there does not seem to be wholesale random ordering when the first field for a view ordering is never or rarely used.
